src/Compilers/Core/Portable/PEWriter/Members.cs

```diff
@@ -415,7 +415,7 @@ ImmutableArray<ExceptionHandlerRegion> ExceptionRegions
         /// <summary>
         /// Debugging information associated with a MoveNext method of a state machine.
         /// </summary>
-        StateMachineMoveNextBodyDebugInfo MoveNextBodyInfo { get; }
+        StateMachineMoveNextBodyDebugInfo? MoveNextBodyInfo { get; }
```

Comment:
>StateMachineMoveNextBodyDebugInfo? MoveNextBodyInfo { get; } [](http://example.com/codeflow?start=8&length=60)

Are changes in this file primarily related to the goal of the PR?

Reply:
I don't remember changing this file, might be wrong merge/rebase, will revert.

Reply:
No but they allow the new code to implement these interfaces without nullability warnings.